packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js

```diff
@@ -62,6 +62,7 @@ export function addItem(
     state: 'PENDING',
     resource: {
       ...resource,
+      originalResourceId: resource?.id,
```

Comment:
```suggestion
      originalResourceId: resource.id,
```
No need for the optional chaining. `resource` is never `null` here.

If `resource` would be `null`, `...resource` above would throw an error.

Reply:
This is needed, unit test failed without it.

Reply:
Can you point to the relevant unit test?

Reply:
Fixed.